utility/src/main/java/org/oppia/android/util/parser/UrlImageParser.kt

```diff
@@ -60,6 +59,10 @@ class UrlImageParser private constructor(
     urlDrawable, { it }
   )
 
+  private inner class OppiaImageTarget(urlDrawable: UrlDrawable) : CustomImageTarget<OppiaImage>(
+    urlDrawable, { }
+  )
+
   private open inner class CustomImageTarget<T>(
```

Comment:
@MohamedMedhat1998 I am stuck here, here we need a way to determine whether `oppiaImage` is a `BitmapImage` or it is `SvgImage` then we can code the lambda function accordingly.

Reply:
you can do something like that:  
```
when (it) {
        is BitmapImage -> {
          
        }
        is SvgImage -> {
          
        }
      }
```

Reply:
@prayutsu I haven't taken a look at the full code in the PR, if there is something that you need me to check, please mention me there